test/Application.UTest/Items/ReforgeUpgradedItemCommandTest.cs

```diff
@@ -181,8 +181,9 @@ public async Task AlreadyOwnedBaseItem()
             UserId = user.Id,
         }, CancellationToken.None);
 
-        var errorCode = result.Errors![0].Code;
-        Assert.That(errorCode, Is.EqualTo(ErrorCode.ItemAlreadyOwned));
+        Assert.That(result.Errors, Is.Null);
+        Assert.That(result.Data?.Item.Id, Is.EqualTo("a_h0"));
+        Assert.That(result.Data?.Item.Rank, Is.EqualTo(0));
     }
```

Comment:
Maybe Check that user owns two a_h0 and check that heirloompoint == 6 ?

Reply:
++